mne/filter.py

```diff
@@ -2622,6 +2630,9 @@ def filter(
                   ``len(picks) * n_times`` additional time points need to
                   be temporarily stored in memory.
 
+        .. note:: When working on SourceEstimates the sample rate of the original
+                    data is inferred from tstep.
+
```

Comment:
Probably not critical enough to be a `.. note::`
```suggestion
        When working on SourceEstimates the sample rate of the original
        data is inferred from tstep.

```